cygwin.rst

```diff
@@ -2,119 +2,134 @@
+  $ln -s /cygdrive/c/Python27/python.exe /usr/bin/python
 
 .. _virtualenv:
 
 Virtual Environment
 -------------------------------
 
-`A virtual environment tool <https://virtualenv.pypa.io/en/stable/userguide/>`_ creates multiple Pythons, each has its packages and dependencies.
- To install virtualenv in Windows, you can use pip command, which is already shipped with Python 3.
-
-.. code-block:: none
-
-  $ pip install virtualenv
+`A virtual environment <https://virtualenv.pypa.io/en/stable/userguide/>`_ is a tool to create isolated Python environments. The basic problem being addressed is one of dependencies and versions, and indirectly permissions. 
 
-Create a new directory for your odkdocs work:
+To start with the installation first create a new directory for your odkdocs work:
 
 .. code-block:: none
 
   $ mkdir odk
+  $ cd odk
     
 You have two options: 
 
   - Use the native virtualenv.
   - Use virtualenvwrapper.
 
-.. _native-virenv:
+.. _virenv-wrapper:
 
-Native Virtual Environment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Virtual Environment Wrapper
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Create a new Python 3 virtual environment, "odkenv" is the name of the virtualenv, you can add any name.
+Dealing with Virtualenv tool might be tiresome in windows. As a simpler alternative, you can install `virtualenvwrapper <https://pypi.python.org/pypi/virtualenvwrapper-win>`_ tool using the following command.
 
 .. code-block:: none
 
-  $ virtualenv -p <python path/python.exe> odkenv
- 
-After creating python3 virtualenv in the previous step, multiple files are copied into the folder odkenv.
+  $ pip install virtualenvwrapper
 
-.. code-block:: none
+Create a new virtualenv use the command:
 
-  $ ls odkenv
+.. code-block:: none
 
-The folder Scripts contains all virtualenv controls as ".bat" files.
+  $ mkvirtualenv odkenv
 
-To activate the odkenv:
+Once the odkenv is created, it is automatically activated, the current path in cmd will appear for example as:
 
 .. code-block:: none
 
-  $ cd odkenv
-  .
-  .
-  .
-  $ cd Scripts
+  $ (odkenv) C:/odk/docs
 
-  $ odk/odkenv/Scripts/activate.bat
+To deactivate the odkenv, write:
 
+.. code-block:: none
 
-To deactivate the odkenv:
+  $ deactivate
+
+To activate the odkenv any time:
 
 .. code-block:: none
 
-  $ odk/odkenv/Scripts/deactivate.bat
+  $ workon odkenv
 
-.. _virenv-wrapper:
+.. _native-virenv:
 
-Virtual Environment Wrapper
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Native Virtual Environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Dealing with Virtualenv tool might be tiresome under windows. For simplicity, you can install `virtualenvwrapper <https://pypi.python.org/pypi/virtualenvwrapper-win>`_ tool using the following command.
+To install virtualenv in Windows, you can use pip command, which is already shipped with Python 3.
```

Comment:
As you mentioned virtualenvwrapper is better when it comes to windows so I have just changed the order of virtualenvwrapper and virtualenv.

Reply:
No, you need to install virtualenv first, then the wrapper.

Reply:
Oh so just the installation commands needs to be there right. Not activate deactivate. Should i include the installation command and commit again?

Reply:
The reason why virtualenvwrapper needs to be first is one goes through all the pain of setting virtualenv first then realizes that it could have been done easily using virtualenvwrapper

Reply:
Did you install the virtualenvwrapper?

Reply:
Yes I did but only after installing virtualenv as it was first in the doc, probably which is why it didn't show any error